### PR TITLE
signup pg

### PR DESCRIPTION
**Pull request by aurobindoGupta**
image

### CODE REVIEW

File: components/signupPg/index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          
+        </div>
+      </div>
+    
+      <!-- ................PAGE CONTENT ............................... -->
+      <!-- ................FOOTER ............................... -->
+
+      <!-- ................FOOTER ............................... -->
+    </div>
+    <!-- ................BASE CONTAINER............. -->
```
Comment (aniketsingh13) [inline components/signupPg/index.html:87]: what is this for?

File: components/signupPg/styles.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
+@import url(/styles.css);
+
+
+.page-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```
Comment (aniketsingh13) [inline components/signupPg/styles.css:7]: you use utility for this

File: components/signupPg/styles.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+.signup-Modal{
+  width: 100%;
+  height: 100%;
+  position: relative;
+}
+.signup-Modal .modal-container{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-evenly;
```
Comment (aniketsingh13) [inline components/signupPg/styles.css:22]: here also

File: components/signupPg/styles.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  height: 25rem;
+  width: 30rem;
+  left: 30%;
+  right: 30%;
+  
+}
+.modal-container .form-input{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```
Comment (aniketsingh13) [inline components/signupPg/styles.css:33]: here also

File: components/signupPg/styles.css [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  width: 100%;
+}
+.form-utils{
+  display: flex;
+  align-items: center;
+}
+.modal-container .modal-btn{
+  width: 70%;
+  display: flex;
+  flex-direction: column;
```
Comment (aniketsingh13) [inline components/signupPg/styles.css:48]: here

**Review by aniketsingh13 — APPROVED** [review]
rest thing look good to me. please consider the changes

File: components/signupPg/index.html (hunk 1 above)
Comment (aurobindoGupta) [reply components/signupPg/index.html:87]: markups for future use...same in all three prjects

**Review by saicharan-gouru — APPROVED** [review]
Looks good to me.

**Comment by arpit01923** [comment]
LGTM but some minor changes said by Aniket must be resolved

**Pull request merged**